api/v1alpha3/machinehealthcheck_webhook.go

```diff
@@ -148,3 +149,7 @@ func (m *MachineHealthCheck) validate(old *MachineHealthCheck) error {
 
 	return apierrors.NewInvalid(GroupVersion.WithKind("MachineHealthCheck").GroupKind(), m.Name, allErrs)
 }
+
+func DisableMinimumNodeStartupTimeout() {
```

Comment:
I suggest to add a comment to this func explaining why we need it.
Also, might be work to add "test" somewhere in the name to make it more explicit this is something that we allow only for tests